tests/test_rbac_api.py

```diff
@@ -33,17 +33,21 @@ def test_add_role_for_user(self):
             get_examples("rbac_model.conf"), get_examples("rbac_policy.csv")
         )
         e.add_role_for_user("alice", "data1_admin")
-        self.assertEqual(e.get_roles_for_user("alice"), ["data2_admin", "data1_admin"])
+        self.assertCountEqual(
```

Comment:
Why change to assertCountEqual? Will this lose accuracy?

Reply:
> #### [assertCountEqual(*first*, *second*, *msg=None*)](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual)
> Test that sequence *first* contains the same elements as *second*, regardless of their order. When they don’t, an error message listing the differences between the sequences will be generated.

In some cases the  new role manager returns the elements in a different order. Now the test only checks, that both lists consist of the same elements.

Reply:
@techoner

Reply:
@abichinger Here you can use `sorted()` first, then compare, or use `assertItemsEqual` to compare.

Reply:
@techoner according to the [documentation](https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertItemsEqual) `assertItemsEqual` was renamed to `assertCountEqual` in Pyhton 3.
I must admit the name of the function `assertCountEqual` is misleading and therefore i will replace all occurrences with `assertEqual(sorted(a), sorted(b))`.

Reply:
@techoner any further comment?